packages/data/src/lookitS3.ts

```diff
@@ -176,6 +187,16 @@ class LookitS3 {
     const timestamp = new Date().toISOString();
     console.log(`Recording log: ${timestamp}\nFile: ${this.key}\n${msg}\n`);
   }
+
+  /**
+   * Whether or not an upload is in progress (created and not yet completed).
+   *
+   * @returns Boolean indicating whether or not an upload has been created but
+   *   not yet completed.
+   */
+  public get uploadInProgress(): boolean {
+    return this.uploadId !== "" && this.complete == false;
+  }
```

Comment:
Do you prefer `this.complete === false` over `!this.complete`? We need to update the `==` to `===`.

Reply:
Nope I prefer your suggestion, thanks!